spaces/Sogur/index.json

```
@@ -0,0 +1,28 @@
+{
+  "key": "Sogur",
+  "name": "Sogur",
+  "chainId": 1,
+  "decimals": 18,
+  "symbol": "SGR",
+  "defaultView": "core",
+  "address": "0xaea8e1b6cb5c05d1dac618551c76bcd578ea3524",
+  "token": "0xaea8e1b6cb5c05d1dac618551c76bcd578ea3524",
+  "core": [
+    "0x3Ff63EC64143993C9eDf89e36f58b9bd69f173AD"
+  ],
+  "min": 10,
```

The minimal balance should be 10 full units (i.e taking into account the 18 decimals). I am not sure if this value should be 10 or 10000000000000000000 so adding the  other option as a suggestion:
```suggestion
  "min": 1010000000000000000000,
```

---

It should be 10